### ISSUE TITLE
Error matching a v-for directive.

### ISSUE THREAD

**Issue by lksnmnn** [testdrivenio/vue-lexer#1]
Hey there,

thank you for this lexer.

I am currently writing my thesis and was looking for a way to make the Latex code listings look nice. I use the minted package which uses pygmentize itself. But so far, the output does not yet satisfy my requirements.

Here is an example code snippet, where the lexer falsely marks `="` and `"` in the second `v-for` loop as an error.

``` vue
<template>
    <svg viewBox="-200, -200, 400, 400">
        <g id="nodes">
            <circle 
                v-for="node in graph.nodes"
                :key="node.id"
                :x="node.x"
                :y="node.y"
                :r="getNodeRadius(node)"
            >
        </g>
        <g id="links">
            <line
                v-for="link in graph.links"
                :key="link.id"
                :x1="link.source.x"
                :x2="link.target.x"
                :y1="link.source.y"
                :y2="link.target.y"
                :style="getLinkStyle(link)"
            />
        </g>
    </svg>
</template>
```
👍 GitHubs Lexer does a good job ;)

The wrong output line looks like this: 
```
<span class="na">v</span><span class="nt">-for</span><span class="err">=&quot;</span><span class="na">link</span> <span class="na">in</span> <span class="na">graph.links</span><span class="err">&quot;</span>
```

![Bildschirmfoto 2019-10-19 um 12 29 03](https://user-images.githubusercontent.com/4983285/67143486-286afa00-f26c-11e9-84fd-43a62fa05942.png)

I attached the full HTML output (as txt, Github doesnt support HTML 😩).
[test.txt](https://github.com/testdrivenio/vue-lexer/files/3746544/test.txt)

Cheers,
Lukas

**Comment by mjhea0** [comment]
Hi Lukas - So, you're using my vue lexer with https://github.com/gpoore/minted and it's not working?

**Comment by lksnmnn** [comment]
Yes. But the above described error is also happening, when I do a plain `pygmentize` call via command line. Like in your examples I call

```shell
pygmentize -O full -f html -o test.html code/test.vue
```

With `test.vue` having the above content.

The lexer is installed: 
```shell
pygmentize -L lexers | grep vue
* vue, vuejs:
    vue (filenames *.vue)
```

Is it working for you?

**Comment by mjhea0** [comment]
You can see the example v-for here: https://github.com/testdrivenio/vue-lexer/blob/master/examples/example3.png and I'm testing a v-for here: https://github.com/testdrivenio/vue-lexer/blob/master/tests/test_lexer.py#L79

I'm not sure why your lexer is not working right. Want to add some test code?

**Comment by lksnmnn** [comment]
Example 3 works correctly on my machine too.
I added a PR with an additional v-for test which should fail. See PR #2
